doggos/inference/takagi_sugeno_inference_system.py

```diff
@@ -1,6 +1,31 @@
+from typing import Dict
+
+from doggos.fuzzy_sets.membership import MembershipDegree
 from doggos.inference.inference_system import InferenceSystem
+from doggos.knowledge import Clause
 
 
 class TakagiSugenoInferenceSystem(InferenceSystem):
-    def output(self) -> float:
+    def output(self, features: Dict[Clause, MembershipDegree], measures: Dict[str, float], method: str = "base") -> float:
```

Comment:
I'd suggest changing method = 'base' to 'weighted average' or 'average'.
Whether measures should be Dict[str, float] or Dict[LinguisticVariable, float] should be discussed.